common/configvar.c

```diff
@@ -4,6 +4,7 @@
 #include <ccan/tal/str/str.h>
 #include <common/configvar.h>
 #include <common/utils.h>
+#include <unistd.h>
```

Comment:
Huh, POSIX says this is in <sys/types.h>, but if this works...

Reply:
yeah, I used this suggestion https://stackoverflow.com/a/65859211/10854225

I think is contained in more than one include (?)